airflow/operators/docker_operator.py

```diff
@@ -71,7 +76,8 @@ def __init__(
             image,
             api_version=None,
             command=None,
-            cpus=1.0,
+            cpus=None,
```

Comment:
I think @mistercrunch was asking you to make this change in a backwards compatible way in your other PR. This looks to me like you've just left the parameter dangling. It doesn't appear to be used anymore. This isn't backwards compatible.